text/0010-gatsby-docs-localization.md

```diff
@@ -0,0 +1,212 @@
+* Popular with translators.
+* We can make an official plugin for Crowdin integration that other projects can use.
+
+However, Crowdin has its fair share of disadvantages as well. We had considered using Crowdin for the localization of the React docs but realized that Crowdin had some [major drawbacks](https://reactjs.org/blog/2019/02/23/is-react-translated-yet.html):
+
+* It has a steep learning curve and hard for new users (especially beginning translators) to get accustomed to.
+* Doesn't handle web markup well; sites translated with Crowdin sometimes have invalid markup
+* Quality of English documentation is mediocre, which is not a good sign for a translation app (see links above).
+
+There are other SaaS platforms such as Transifex that fix some of Crowdin's issues. However, the core issue relies in that they are new systems that most developers are unfamiliar with. We are relying on developers from the Gatsby community rather than professional translators, and those developers tend to be more familiar with GitHub's UI.
+
+### Other GitHub-based arrangements
+
+The monorepo isn't the only way to organize our code. We could do something like React and have each language be in a separate repo copy. This makes it easier to copy content and check for changes, since we can just use `git merge`.
+
+But this has its own drawbacks as well. Each language is its own website and we can't easily switch between languages. The rest of Gatsby's code is in a monorepo and has tooling to make it work, so it makes sense to continue with that paradigm. We'd also like to dogfood Gatsby's localization utilities such as [`gatsby-plugin-i18n`](https://www.gatsbyjs.org/docs/localization-i18n/#gatsby-plugin-i18n).
+
+## Adoption Strategy
+
+To test-drive the process, we can start with three languages or so, based on user interest in this RFC. We will have to manually set up those translations while we create the automation, but they should provide a test bed to make sure our checks work.
+
+The only thing we need to do before those languages are set up is move content over to `/docs/en` and handle the routing appropriately. Once that's done, the translations can start and we can divide up the other changes needed for the website.
+
+Once the automation work for setting up new translations is done, we can open up pull requests to create new language to more people.
+
+When the first languages have a "core" set of pages translated (for example, the Home Page, Tutorial, and API Reference), that translation can be marked as "Complete".
+
+
+## How will we teach this
+
+We'll create additional documentation in both the monorepo and the site itself on the process of contributing translations. There should be documentation for:
+
+* How to contribute a new translation (with a link to the progress issue)
+* How the automation works and what to expect
+* The responsibilities of a translation maintainer
+* Tips for translators
```

Comment:
Here as well I can think of a lot of things that happened in the Spanish docs for React 😅

Reply:
hey @Darking360 !

me and others started translating the docs to spanish [here](https://github.com/horacioh/gatsby)

Happy to chat and get some help from your experience!

Reply:
Hey, @horacioh sure! I was hoping to contact you since you have work already started and it's a super good starting point, just checked the project this afternoon and looks really good 😱  and a lot of folks can jump into the official documentation from there too, I'm going to reach you by your email and we can start setting up the ground for this 💪

Reply:
hell yeah! 🚀 
hit me at hi[at]horacioh[dot]com. thanks for your help!!